### ISSUE TITLE
APITesting - request not defined

### ISSUE THREAD

**Issue by DennisRein** [SAP/ui5-uiveri5#108]
I tried to run api tests like described in the sample/apiTesting/APITesting.spec.js.
So I created the APITesting.spec.js file containing of:
```javascript
describe('APITesting', function() {
    it('Should make api call to / and check reponse body', function() {
        request.get(browser.testrunner.config.params.apiURL+'/').do()
            .catch(function(responseError) {
                expect(responseError.status).toBe(401);
            });
    });
});

```
As soon as I run uiveri5 I get following error:
```
INFO: Suite started: APITesting
INFO: Spec started: Should make api call to / and check reponse body
INFO: Opening: https://baseurl.com/
INFO: UI5 Version: 1.70.0-SNAPSHOT
INFO: UI5 Timestamp: 201908230515
INFO: Expectation FAILED: Failed: request is not defined
INFO: Spec finished: Should make api call to / and check reponse body with status: FAILED
```

I don't think I need to import request into my spec file or do I? 
What am I missing here?

Best regards
Dennis

**Comment by maximnaidenov** [comment]
Hi, did you define the proper profile in conf.js ?
Here is our docs (seems missing this config step but you can have a look at the sample): https://github.com/SAP/ui5-uiveri5/blob/master/docs/usage/api.md

**Comment by LukasHeimann** [comment]
Hi @maximnaidenov, thank you for your quick response!
So it is not possible to use `uiveri5` to test both UI integration and APIs at the same time with one `conf.js`? This would be very useful when performing automated testing in Pipelines e.g. with Piper (https://github.com/SAP/jenkins-library), which starts UIVeri5 once...
Kind regards
Lukas

**Comment by maximnaidenov** [comment]
Hi Likas,
of course it is possible to use API testing in UI tests, that is the idea - prepare content before the actual test or validate results with API calls if the UI does not show the respective objects. API profile extends the integration profile e.g. adds the configuration. If you want, you can have a look at the api.profile.conf.js and just copy the configurations to your existing conf.js. 
We will need to improve the documentation for sure.

**Comment by maximnaidenov** [comment]
Hi Lukas, did you manage to use the request object ?. I am really interested in your feedback as you would be our first user for this feature. The team that requested it did back-off at some point.
Please do not hesitate to ask any other questions you may have here.

**Comment by lboehm** [comment]
Hi @maximnaidenov, 

I've got the same problem as @LukasHeimann. When running my spec, it fails with error message "request is not defined".
Then I found this issue were you wrote in [this comment](https://github.com/SAP/ui5-uiveri5/issues/108#issuecomment-524328788) that conf.js should be adapted. I assume you talked about [this sample](https://github.com/SAP/ui5-uiveri5/blob/master/sample/apiTesting/conf.js).
So profile should be `api`, right?

So my question is: if i have on conf.js for all my uiveri5 tests, how should I modify my conf.js so I can run `api` amd `integration` tests at once?

Thanks for your answer!
Lukas

**Comment by maximnaidenov** [comment]
@lboehm Api profile inherits all configs from Integration profile so simply setting it should work fine. The missing specResolver config was a bug, we fixed some time ago but seems forgot to release a version. I will do so shortly.

**Comment by lboehm** [comment]
@maximnaidenov Perfect, thanks a lot!

**Comment by tsaleksandrova** [comment]
fixed with #125 and #152
